### ISSUE TITLE
Feature request. World Generation per dimension

### ISSUE THREAD

**Issue by gollariel** [P3pp3rF1y/AncientWarfare2#906]
Would be nice to have a whitelist of dimensions to enable world generation. First of all it useful if you already have the world, also it can help like me when I want to have the generation, but it looks strange with quark_realistic and traverse. (For example when the house of woodcutter generates in the center of the lake :))

**Comment by MohawkyMagoo** [comment]
Dimension whitelist already exists - you need to set dimension whitelist to true in the AW structure template that you want to spawn and thenenter the dimesion ID. 

On the other issue - Unfortunately some biome mods use large areas of water which they dont define as "water" - this includes BOP, Quark and Traverse - so AW structures have no way of knowing what is water and what is land in these biomes - you can test this by flying over the "lake" in creative and see what biome it gives you.

**Comment by gollariel** [comment]
I miss Dimension whitelist. I see... so will be better to use standard generation for ancient warfare... Thanks a lot!

**Comment by gollariel** [comment]
hm. I checked the templates.... If I understand correctly I can not just add whitelisted dimension, It always must be changed in each .aws file. Previously disabled default template file... Looks like very complex thing... Also, I not sure where can I get default file? I downloaded mohawky_structure_pack directory... 

I think my request can still be here. Obviously will be easier to change dimension in one config file
